src/viam/ur/module/ur_arm.cpp

```diff
@@ -457,6 +457,18 @@ void URArm::configure_(const std::unique_lock<std::shared_mutex>& lock, const De
     ur_cfg.headless_mode = true;
     ur_cfg.socket_reconnect_attempts = 1;
 
+    if (!current_ports_) {
+        current_ports_ = new_ports();
+    }
+    ur_cfg.reverse_port = current_ports_->reverse_port;
+    ur_cfg.script_sender_port = current_ports_->script_sender_port;
+    ur_cfg.trajectory_port = current_ports_->trajectory_port;
+    ur_cfg.script_command_port = current_ports_->script_command_port;
+    VIAM_SDK_LOG(debug) << "using reverse_port " << ur_cfg.reverse_port;
+    VIAM_SDK_LOG(debug) << "using script_sender_port " << ur_cfg.script_sender_port;
+    VIAM_SDK_LOG(debug) << "using trajectory_port " << ur_cfg.trajectory_port;
+    VIAM_SDK_LOG(debug) << "using script_command_port " << ur_cfg.script_command_port;
```

Comment:
if these do not show up in the logs when debug is configured, go ahead and change them to info level